most.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const most = require('most');
+const R = require('ramda');
+const S = require('sanctuary');
+
+
+//    join :: String -> String -> String
+const join = R.curryN(2, path.join);
+
+//    readFile :: String -> String -> Promise Error String
+const readFile = R.curry((encoding, filename) =>
+  new Promise((resolve, reject) =>
+    fs.readFile(filename, { encoding }, (err, data) =>
```

Comment:
Let's use `{encoding: encoding}` here. I'd like to keep the code compatible with Node 0.12.x.